src/rgw/rgw_exporter.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <atomic>
+#include <thread>
+#include <string>
+
+#include "rgw_usage_cache.h"
+#include "rgw_perf_counters.h"
+class CephContext;
+
+namespace rgw{
```

Comment:
nit, missing space.

Reply:
Oh yes, the more important thing, include `ceph_context.h` instead of using the forward declaration, as it breaks compilation.